<launch>
    <node pkg="joy" type="joy_node" name="joy_node" required="true" output = "screen" />
    <param name="linear_x_max" value="1.5" />
    <param name="linear_x_min" value="0.05" />
    <param name="linear_y_max" value="1.5" />
    <param name="linear_y_min" value="0.05" />
    <param name="angular_max" value="1.5" />
    <param name="angular_min" value="0.1" />
    <param name="acceleration" value="3." />
    <param name="deceleration" value="3." />

    <param name="topic_joy" value="joy"/>
    <param name="topic_cmdvel" value="/swerve_steering_controller/cmd_vel"/>
    <param name="topic_enablePS3" value="control_ps3"/>

    <node pkg="ps3joy" type="controlPS3_swerve.py" name="ps3" output = "screen" />
  </launch>
  